manifests/charts/gateways/istio-egress/templates/autoscale.yaml

```diff
@@ -22,6 +23,34 @@ spec:
     - type: Resource
       resource:
         name: cpu
-        targetAverageUtilization: {{ $gateway.cpu.targetAverageUtilization }}
+        target:
+          type: Utilization
+          averageUtilization: {{ $gateway.cpu.targetAverageUtilization }}
 ---
+{{- else }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ $gateway.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+  {{ $gateway.labels | toYaml | indent 4 }}
+release: {{ .Release.Name }}
```

Comment:
indent is off.

Since the schema is identical can we just template the `apiVersion` part?

Reply:
Two templates are different, check the resource section, they are slightly off